packages/documentation/package.json

```
@@ -1,47 +1,26 @@
 {
-  "name": "documentation",
-  "version": "0.0.0",
-  "private": true,
+  "name": "rafiki-starlight",
```

```suggestion
  "name": "documentation",
```
This allows us to do `pnpm --filter documentation <command>` from the root of the repo